### PR TITLE
Add cop to detect old Berkfile sources

### PR DESCRIPTION
**Pull request by tas50**
Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

File: lib/rubocop/cop/chef/berksfile_source.rb [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+module RuboCop
+  module Cop
+    module Chef
+      # Over the coursee of years there have been many URLs to use in a cookbook's Berksfile
```
Comment (tyler-ball) [inline lib/rubocop/cop/chef/berksfile_source.rb]: `coursee` -> `course`

File: lib/rubocop/cop/chef/berksfile_source.rb [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      # These old
```
Comment (tyler-ball) [inline lib/rubocop/cop/chef/berksfile_source.rb]: Incomplete sentence

File: lib/rubocop/cop/chef/berksfile_source.rb (hunk 1 above)
Comment (tas50) [reply lib/rubocop/cop/chef/berksfile_source.rb]: Fixed

File: lib/rubocop/cop/chef/berksfile_source.rb (hunk 2 above)
Comment (tas50) [reply lib/rubocop/cop/chef/berksfile_source.rb]: Fixed

**Pull request merged**